Index: CMakeLists.txt
===================================================================
--- CMakeLists.txt	(revision 1084250)
+++ CMakeLists.txt	(working copy)
@@ -1,3 +1,14 @@
+project(solid-netwworkmanager-0.7)                                        
+
+find_package(KDE4 REQUIRED)
+include (KDE4Defaults)     
+include(MacroOptionalAddSubdirectory)
+
+add_definitions (${QT_DEFINITIONS} ${KDE4_DEFINITIONS})
+include_directories (${KDE4_INCLUDES})    
+
+macro_optional_find_package(NetworkManager)
+
 include_directories(
    ${KDEBASE_WORKSPACE_SOURCE_DIR}/libs
    ${NETWORKMANAGER_INCLUDE_DIRS}
